packages/module/src/MessageBox/MessageBox.tsx

```diff
@@ -17,6 +17,10 @@ export interface MessageBoxProps extends React.HTMLProps<HTMLDivElement> {
   innerRef?: React.Ref<HTMLDivElement>;
   /** Modifier that controls how content in MessageBox is positioned within the container */
   position?: 'top' | 'bottom';
+  /** Custom click handler for when scroll to top jump button is clicked */
+  onScrollToTopClick?: () => void;
+  /** Custom click handler for when scroll to bottom jump button is clicked */
```

Comment:
More a nit and might just be me, but for the descriptions of these maybe instead of "Custom click handler...", something like "Click handler for additional logic for when..."?

Custom click handler sorta comes off to me as it will override the default behavior rather than this being ran in addition to that default click behavior.

Otherwise this lgtm

Reply:
Updated! Erin's out so I can't get her take.